examples/get-started/hello_world/main/hello_world_main.c

```diff
@@ -34,7 +34,7 @@ void app_main(void)
         return;
     }
 
-    printf("%uMB %s flash\n", flash_size / (1024 * 1024),
+    printf("%luMB %s flash\n", flash_size / (uint32_t)(1024 * 1024),
```

Comment:
```suggestion
    printf("%" PRIu32 "MB %s flash\n", flash_size / (uint32_t)(1024 * 1024),
```
(plus `#include <inttypes.h>` at the top of the file)